### DISCUSSION TITLE
Nether Biomes and leaf decay

### DISCUSSION THREAD

**Discussion by PacificDriver** [BG-Software-LLC/SuperiorSkyblock2#1558] [Pending Feature Requests] [answered]
1) A duplicate of the biome menu, only unavailable until nether is unlocked.

2) island setting that allows you to block or enable leaf decay

**Comment by Lemvs** [comment, marked as answer]
1. Implemented in latest dev build.
2. It has already existed for a long time.

@OmerBenGera you can mark this as answered and close it.
